### PR TITLE
Implemented the course details page with mock data

### PR DESCRIPTION
**Pull request by ahamed**
localhost_10008_wp-admin_admin php_page=order-details
- Mock data will be replaced after the API is created

### CODE REVIEW

File: assets/react/v3/entries/order-details/atoms/Badge.tsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
+import { borderRadius, colorTokens, spacing } from '@Config/styles';
```
Comment (sazedul-haque) [inline assets/react/v3/entries/order-details/atoms/Badge.tsx:1]: Coupon has the same kind of badge. So, we should add this in shared directory.

**Review by sazedul-haque — APPROVED** [review]

File: assets/react/v3/entries/order-details/components/order/Payment.tsx [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+            <div>3 Items</div>
+            <div>$1300.00</div>
+          </div>
+          <div css={styles.item({ action: 'regular' })}>
+            <button
+              type="button"
+              css={styles.discountButton}
+              onClick={() => alert('@TODO: Will be implemented later.')}
+            >
+              Add discount
```
Comment (sifat009) [inline assets/react/v3/entries/order-details/components/order/Payment.tsx:32]: need to implement multilingual string

File: assets/react/v3/entries/order-details/components/order/Payment.tsx [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+            </button>
+            <div>-</div>
+            <div>-$0.00</div>
+          </div>
+          <div css={styles.item({ action: 'regular' })}>
+            <div>Estimated tax</div>
```
Comment (sifat009) [inline assets/react/v3/entries/order-details/components/order/Payment.tsx:38]: need to implement multilingual string

File: assets/react/v3/entries/order-details/components/order/Payment.tsx [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+            <div>15%</div>
+            <div>$23.00</div>
+          </div>
+          <div css={styles.item({ action: 'bold' })}>
+            <div>Total Paid</div>
```
Comment (sifat009) [inline assets/react/v3/entries/order-details/components/order/Payment.tsx:43]: need to implement multilingual string

File: assets/react/v3/entries/order-details/components/order/Payment.tsx [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+            <div>Reason: Manual refund</div>
+            <div>-$500.00</div>
+          </div>
+          <div css={styles.item({ action: 'destructive' })}>
+            <div />
+            <div>Reason: -</div>
+            <div>-$500.00</div>
+          </div>
+          <div css={styles.item({ action: 'bold' })}>
+            <div>Net payment</div>
```
Comment (sifat009) [inline assets/react/v3/entries/order-details/components/order/Payment.tsx:59]: need to implement multilingual string

File: assets/react/v3/entries/order-details/components/order/Payment.tsx [hunk 6]
```diff
@@ -0,0 +1,138 @@
...
+          </div>
+          <div css={styles.separator} />
+          <div css={styles.item({ action: 'destructive' })}>
+            <div>Refunded</div>
+            <div>Reason: Manual refund</div>
+            <div>-$500.00</div>
+          </div>
+          <div css={styles.item({ action: 'destructive' })}>
+            <div />
+            <div>Reason: -</div>
```
Comment (sifat009) [inline assets/react/v3/entries/order-details/components/order/Payment.tsx:55]: need to implement multilingual string

File: assets/react/v3/entries/order-details/components/order/Payment.tsx [hunk 7]
```diff
@@ -0,0 +1,138 @@
...
+            <div>$23.00</div>
+          </div>
+          <div css={styles.item({ action: 'bold' })}>
+            <div>Total Paid</div>
+            <div />
+            <div>$1323.00</div>
+          </div>
+          <div css={styles.separator} />
+          <div css={styles.item({ action: 'destructive' })}>
+            <div>Refunded</div>
```
Comment (sifat009) [inline assets/react/v3/entries/order-details/components/order/Payment.tsx:49]: need to implement multilingual string

File: assets/react/v3/entries/order-details/components/order/Activities.tsx [hunk 8]
```diff
@@ -0,0 +1,148 @@
...
+                render={(props) => (
+                  <FormTextareaInput
+                    {...props}
+                    label={__('Add a comment (Only you and other staff can see comments)', 'tutor')}
+                    rows={5}
+                  />
+                )}
+              />
+              <Button type="submit" variant="primary" size="small" isOutlined>
+                Post
```
Comment (sifat009) [inline assets/react/v3/entries/order-details/components/order/Activities.tsx:46]: need to implement multilingual string

File: assets/react/v3/shared/components/Container.tsx [hunk 9]
```diff
@@ -0,0 +1,18 @@
+import { css } from "@emotion/react";
+import { ReactNode } from "react";
+
+const CONTAINER_WIDTH = 1116;
```
Comment (sifat009) [inline assets/react/v3/shared/components/Container.tsx:4]: could we implement something which will change according to the active device breakpoint?

**Review by sifat009 — APPROVED** [review]

File: assets/react/v3/entries/order-details/components/order/OrderItem.tsx [hunk 10]
```diff
@@ -0,0 +1,93 @@
...
+            {(discount) => (
+              <div css={styles.discount}>
+                <SVGIcon name="tagOutline" width={16} height={16} />
+                <p>{discount.name}</p>
+                <p>(-{discount.value})</p>
+              </div>
+            )}
+          </Show>
+
+          {/* {item.type === 'bundle' && (
```
[the remark this answers is not in the document]
Comment (ahamed) [reply assets/react/v3/entries/order-details/components/order/OrderItem.tsx:34]: We have some decision issues. If we need to show the course count for bundle type we need this code.

File: assets/react/v3/shared/components/Container.tsx (hunk 9 above)
Comment (ahamed) [reply assets/react/v3/shared/components/Container.tsx:4]: Not yet think about responsiveness.

**Pull request merged**